### PR TITLE
Ft enable social auth #142465473

### PR DESCRIPTION
**Pull request by Andela-Didacus**
#### What does this PR do?
Enables social media login via google, Facebook and twitter. 
#### Description of Task to be completed?
Integrate the login feature with social logins, google, Facebook and Twitter.
#### How should this be manually tested?
Manual login and logout through the login links.

### CODE REVIEW

File: wger/core/templates/user/login.html [hunk 1]
```diff
@@ -5,15 +5,26 @@
 
 
 {% block header %}
+<link rel="stylesheet" href="static/css/font-awesome.min.css">
```
Comment (osule) [inline wger/core/templates/user/login.html]: This is included in base template from which this extends from.

File: wger/core/templates/user/login.html [hunk 2]
```diff
@@ -5,15 +5,26 @@
...
 {% endblock %}
 
 {% block content %}
+<link rel="stylesheet" href="static/css/font-awesome.min.css">
```
Comment (osule) [inline wger/core/templates/user/login.html]: Same comment as above here.

File: wger/core/templates/user/login.html [hunk 3]
```diff
@@ -5,15 +5,26 @@
...
+
 <form action="{% url 'core:user:login' %}{% if next %}?next={{next}}{% endif %}"
       method="post"
       class="form-horizontal">
     {% trans 'Login' as submit_text %}
     {% render_form_fields form submit_text %}
 </form>
+<p style="margin-top: 1em; margin-left:14em;">
```
Comment (osule) [inline wger/core/templates/user/login.html]: Is this responsive?

File: wger/core/templates/user/login.html (hunk 1 above)
Comment (NgangaMaryanne) [inline wger/core/templates/user/login.html]: removed

File: wger/core/templates/user/login.html (hunk 2 above)
Comment (NgangaMaryanne) [inline wger/core/templates/user/login.html]: removed

File: wger/core/templates/user/login.html (hunk 3 above)
Comment (NgangaMaryanne) [inline wger/core/templates/user/login.html]: yes this works. without it, the buttons are too much to the left margin

**Pull request merged**